notify/impl.go

```diff
@@ -682,6 +688,72 @@ func (n *OpsGenie) Notify(ctx context.Context, as ...*types.Alert) error {
 	return nil
 }
 
+// Pushover implements a Notifier for Pushover notifications.
+type Pushover struct {
+	conf *config.PushoverConfig
+	tmpl *template.Template
+}
+
+// NewPushover returns a new Pushover notifier.
+func NewPushover(c *config.PushoverConfig, t *template.Template) *Pushover {
+	return &Pushover{conf: c, tmpl: t}
+}
+
+func (*Pushover) name() string { return "pushover" }
+
+// Notify implements the Notifier interface.
+func (n *Pushover) Notify(ctx context.Context, as ...*types.Alert) error {
+	key, ok := GroupKey(ctx)
+	if !ok {
+		return fmt.Errorf("group key missing")
+	}
+	data := n.tmpl.Data(receiver(ctx), groupLabels(ctx), as...)
+
+	log.With("incident", key).Debugln("notifying Pushover")
+
+	var err error
+	tmpl := tmplText(n.tmpl, data, &err)
+
+	parameters := url.Values{}
+	parameters.Add("token", n.conf.Token)
+	parameters.Add("user", n.conf.UserKey)
+	title := tmpl(n.conf.Title)
+	message := tmpl(n.conf.Message)
+	parameters.Add("title", title)
+	if len(title)+len(message) > 512 {
```

Comment:
This is quite little space, so that listing all alerts doesn't make sense once you've more than 1 or 2 - and for the defaults we need to consider the case where there could be hundreds. Could you cut this down to say only the labels in the message?